### PR TITLE
Remove namespace creation in helm install doc

### PR DESCRIPTION
**Pull request by lahabana**
The namespace is now created by the helm chart

Signed-off-by: Charly Molter <charly.molter@konghq.com>

### CODE REVIEW

**Review by austince — APPROVED** [review]

**Comment by austince** [comment]
Wait, sorry – where is it created in the Helm chart? All I see is the patch: https://github.com/kumahq/kuma/blob/master/deployments/charts/kuma/templates/pre-install-patch-namespace-job.yaml

File: docs/docs/1.2.3/installation/helm.md [hunk 1]
```diff
@@ -27,7 +27,6 @@ Once the repo is added, all following updates can be fetched with `helm repo upd
 At this point we can install and run Kuma using the following commands. We could use any Kubernetes namespace to install Kuma, by default we suggest using `kuma-system`:
 
 ```sh
-$ kubectl create namespace kuma-system
 $ helm install --namespace kuma-system kuma kuma/kuma
```
Comment (austince) [inline docs/docs/1.2.3/installation/helm.md:30]:
```suggestion
$ helm install --create-namespace --namespace kuma-system kuma kuma/kuma
```
Maybe this is what we want instead?

Comment (lahabana) [reply docs/docs/1.2.3/installation/helm.md:30]: This seems correct indeed.
Seems like I misunderstood something when we talked about it with @bartsmykla.
@bartsmykla  can you double check that what @austince is saying matches what you said earlier?

**Comment by Bradamant3** [comment]
Added do not merge label until @bartsmykla reviews -- then you can remove and I'll give it some writer love if needed.

File: docs/docs/1.2.3/installation/helm.md (hunk 1 above)
Comment (bartsmykla) [reply docs/docs/1.2.3/installation/helm.md:30]: it's exactly what I was thinking about

**Review by bartsmykla — APPROVED** [review]
lgtm

**Pull request merged**